### DISCUSSION TITLE
Сборка, Highlight

### DISCUSSION THREAD

**Discussion by DM-VLK** [shmuz/far2m#15] [Ideas]
Спасибо вам за правки и добавление goto! Пересобрал обновленные версии и все заработало (кроме плагина Highlight), только luarocks ругался на отсутствие lua51, но установка соответствующего пакета решила проблему. Жаль, что я не очень в ладах с lua чтобы тестировать его работу. Также есть немного вопросов:

1.	Возможно, стоит пока исключить из файла сборки плагин Highlight раз он еще не готов?
2.	Сделаете ли вы в будущем чтобы сборка плагинов не требовала команды ln -s /usr/local/lib/far2l/Plugins/luafar/luafar2l.so . ?
3.	А почему бы не слить воедино ваш far2l и плагины раз и то и другое разрабатывается вами?

**Comment by shmuz** [comment]
Извиняюсь за задержку с ответом (почему-то был уверен, что приходят уведомления на почту, но видимо это надо сконфигурировать).

1. Плагин Highlight работает, и я им пользуюсь, но он на данный момент далеко не конкурент Colorer'у, поэтому я предпочитаю, чтобы пользователи работали с колорером. Мне не хочется его убирать именно потому, что я с ним работаю. 
2. Уже думал об этом, пока знаний не хватает. (Кстати у меня линк на
`~/far2l/_build/install/Plugins/luafar/luafar2l.so` хотя это наверное неважно).
3. Пока ещё "не созрел" для этого. Поскольку пользователей нет, трудновато определять, что лучше, что хуже.

Буду благодарен за обратную связь, репорты и т.п.

P.S. Поправил Makefile, теперь Highlight будет собираться только если $USER=shmuel

**Comment by shmuz** [comment]
Для информации:
1. Плагин Polygon больше не нуждается в lsqlite3 (но по-прежнему нуждается в sqlite3)
2. При установке Lua-библиотек с помощью luarocks надо указывать в конце не `--lua-version 5.1` как я вам советовал,
а `--lua-version luajit`, тогда не пришлось бы устанавливать Lua 5.1

**Reply by DM-VLK** [reply]
Спасибо
